kpops/component_handlers/topic/handler.py

```diff
@@ -150,7 +149,7 @@ def __check_partition_count(
             log.error(
                 f"Topic Creation: partition count of topic {topic_name} changed! Partitions count of topic {topic_name} is {partition_count}. The given partitions count {topic_spec.partitions_count}."
             )
-            sys.exit(1)  # FIXME raise instead https://github.com/bakdata/kpops/issues/101
+            raise ValueError(f"Error creating the topic {topic_name}")
```

Comment:
Use the message of the log in the Error and remove the log

Reply:
additionally we should consider introducing custom Exception types

Reply:
@disrupted Agree! We should create a Issue for that to track it!

Reply:
maybe now is better (?)

Reply:
perhaps you could resolve #215 in this PR if it's not too much effort?

Reply:
You mean all the exceptions? But let me do it in a separé PR I think it makes more sense. This will merged on the windows part